source/common/init/target_impl.cc

```diff
@@ -35,7 +35,7 @@ absl::string_view TargetImpl::name() const { return name_; }
 
 TargetHandlePtr TargetImpl::createHandle(absl::string_view handle_name) const {
   // Note: can't use std::make_unique here because TargetHandleImpl ctor is private.
-  return std::unique_ptr<TargetHandle>(
+  return TargetHandlePtr(
```

Comment:
Oh, nice, I should have done this to start with :)

Reply:
We will gradually move the ideal shape after iterations!